collectors/0/samza_eet.py

```diff
@@ -0,0 +1,23 @@
+#!/usr/bin/python
+import sys
+
+from collectors.lib.samza_metric_reporter import SamzaMetricReporter
+
+CONSUMER_GROUP_ID = "tcollector_samza_eet"
+
+KAFKA_BOOTSTRAP_SERVERS = [
+    "1.kafka-eet.us-east-1.backend-staging.optimizely:9094",
+    "2.kafka-eet.us-east-1.backend-staging.optimizely:9094",
+    "3.kafka-eet.us-east-1.backend-staging.optimizely:9094",
+    "4.kafka-eet.us-east-1.backend-staging.optimizely:9094",
+    "5.kafka-eet.us-east-1.backend-staging.optimizely:9094",
+]
+
+
+def main():
+
+    reporter = SamzaMetricReporter(CONSUMER_GROUP_ID, KAFKA_BOOTSTRAP_SERVERS, 'eet.')
```

Comment:
eet-staging? also name the file `eet-staging` since we'll eventually have prod?

Reply:
@vraja2 done